src/python/qequlacs/utils.py

```diff
@@ -1,5 +1,15 @@
+"""
+Functions to convert z-quantum-core circuits to Qulacs.
+
+Approach:
+ * base is QuantumCircuit
+ * goes over gates one-by-one
+ * corresponding Qulacs gates appended to circuit
+ * if no matching gate on Qulacs side - unitary matrix used
+ * for > 3 qubits exception is raised (not implemented yet)
+"""
+
 import qulacs
-from zquantum.core.circuit import Circuit, Gate
 import numpy as np
 
 def convert_circuit_to_qulacs(circuit):
```

Comment:
I'm going to be a stickler here and ask for proper docstrings on all the functions in this file.

Reply:
agree